mono/utils/mono-merp.h

```diff
@@ -0,0 +1,51 @@
+/**
+ * \file
+ * Support for interop with the Microsoft Error Reporting tool (header)
+ *
+ * Author:
+ *   Alexander Kyte (alkyte@microsoft.com)
+ *
+ * (C) 2017 Microsoft, Inc.
+ *
+ */
+#ifndef __MONO_UTILS_MERP__
+#define __MONO_UTILS_MERP__
+
+#include <config.h>
+#include <glib.h>
+
+#ifdef HOST_DARWIN
+
+/**
+ * Unregister the MERP-based handler
+ */
+void mono_merp_disable (void);
+
+/**
+ *
+ * Enable the MERP-based handler and set application-specific information
+ *
+ * See MERP documentation for information on the bundle ID, signature, and version fields
+ */
+void
+mono_merp_enable (const char *appBundleID, const char *appSignature, const char *appVersion, const char *merpGUIPath);
```

Comment:
1. Is it ok to call this more than once?
2. It takes ownership of its argument strings, I don't think they should be 'const'.